docs/config-explain/ingester-config.md

```diff
@@ -0,0 +1,32 @@
+Below is the loki part of a Loki sample config:
```

Comment:
I think this should be part of the readme in loki helm chart, which we don't have yet. Are you willing to take that ? I'm happy to help you build that out.

Reply:
here is an example: https://github.com/helm/charts/blob/master/stable/cert-manager/README.md